SampleBrowser/SampleBrowser/UI/Browser/BrowserViewController.swift

```diff
@@ -140,6 +159,10 @@ class BrowserViewController: UIViewController,
         // If the Client needs to save a title like saving it inside some tab storage then it would do it here
     }
 
+    func onHasOnlySecureContentChanged(secure: Bool) {
+        // If the client needs to show a Secure lock icon etc.
```

Comment:
@thatswinnie Just an FYI, for URL bar project! So you get a sense of how things are moving in the SampleBrowser